lib/menubar/init.lua

```diff
@@ -160,7 +159,8 @@ local function label(o)
     return colortext(gstring.xml_escape(o.name), fg_color),
            bg_color,
            nil,
-           o.icon
+           o.icon,
+           o.icon and {icon_size=instance.geometry.height}
```

Comment:
Can you add that to the function's docs as well?

Reply:
added